lib/views/home/home_screen.dart

```diff
@@ -7,10 +7,10 @@ import 'package:thc/models/enum_widget.dart';
 import 'package:thc/models/local_storage.dart';
 import 'package:thc/models/user.dart';
 import 'package:thc/views/create_livestream/create_livestream.dart';
-import 'package:thc/views/schedule/schedule.dart';
 import 'package:thc/views/manage_surveys/manage_surveys.dart';
 import 'package:thc/views/manage_users/manage_users.dart';
 import 'package:thc/views/profile/profile.dart';
+import 'package:thc/views/schedule/schedule.dart';
```

Comment:
This change was just because a static analysis message was telling me to "sort imports alphabetically".